src/mame/korg/korgtriton.cpp

```diff
@@ -0,0 +1,210 @@
+// license:BSD-3-Clause
+// copyright-holders:Antonio "Willy" Malara
+
+/*
+Korg TRITON skeleton driver
+
+CPU: SH2 SH7043 / SH7045
+FLASH: 2 * MX29F1610
+LCD controller: M66271FP
+Floppy disk controller: HD63266
+Serial Interface: uPD71051
+SCSI controller: MB86604L
+Sound: 2 * "TGL96" MB87F1710-PFV-S (proprietary)
+
+This driver runs the embedded firmware up until it needs to
+relocate and run the main application. The screen remains
+blank for all this time.
+*/
+
+
+#include "emu.h"
+#include "screen.h"
+
+#include "cpu/sh/sh7042.h"
+#include "machine/i8251.h"
+
+namespace {
+
+    class korgtriton_state : public driver_device
+    {
+    public:
```

Comment:
Please don’t indent for namespaces, particularly when it’s the whole file like this.